moz_sql_parser/__init__.py

```diff
@@ -32,6 +32,7 @@ def __deploy__():
 
 def parse(sql):
     try:
+        sql = sql.rstrip().restrip(";")
```

Comment:
the tests failed (example https://travis-ci.org/mozilla/moz-sql-parser/jobs/499841236) .  Notice the spelling mistake: `restrip`

Reply:
Sorry for the typo, wouldn't happen again :)

Updated